aws/resource_aws_mq_broker_test.go

```diff
@@ -1115,16 +1154,76 @@ resource "aws_mq_broker" "test" {
   engine_type        = "ActiveMQ"
   engine_version     = "5.15.0"
   host_instance_type = "mq.t2.micro"
-  security_groups    = ["${aws_security_group.test.id}"]
+	security_groups    = ["${aws_security_group.test.id}"]
+
+	user {
```

Comment:
Had to add `user` on `testAccMqBrokerConfig_updateTags3` as the test was throwing the error `"user": required field is not set` - I assume this is because `user` is actually a required attribute: https://github.com/terraform-providers/terraform-provider-aws/blob/e94c64a/aws/resource_aws_mq_broker.go#L177

Reply:
yeah sorry my bad. My editor keeps defaulting to tabs as the code is in go but the literals are terraform which uses spaces. Will fix!

Reply:
This has been fixed. It was indeed my editor. No diff once I sorted out or tests faling anymore. Thanks!